EIPS/eip-3416.md

```diff
@@ -0,0 +1,114 @@
+
+## Test Cases
+
+## Security Considerations
+
+* Senders cannot manipulate the minimum fee because the minimum `BASE_FEE` is controlled by the miners with small increments or decrements on each new block proposed.
+* Above the `BASE_FEE` the senders have a very limited ability to manipulate and lower the final gas price they pay because they have to move the weighted median close to `BASE_FEE` and, as we know, this is a very robust statistic.
+* Miners have a very limited ability to manipulate and raise the final gas price paid by senders above `BASE_FEE` because to influence the final gas price they have to stuff fake transactions beyond the top 5% of the blocks. In average and currently, only the top 3% of the block is empty, so to fill-up 5% of the block they need to start dropping profitable transactions to reach 5%. Only beyond 5% of the top block gas they can start moving the median a little and the median is still a very robust statistic, not liable to being easily manipulated.
+
+## Resources
+
+* [Call notes](https://github.com/ethereum/pm/blob/master/All%20Core%20Devs%20Meetings/Meeting%2077.md)
+* [Original Magicians thread](https://ethereum-magicians.org/t/eip-1559-fee-market-change-for-eth-1-0-chain/2783)
+* [Ethresear.ch Post w/ Vitalik’s Paper](https://ethresear.ch/t/draft-position-paper-on-resource-pricing/2838)
+* [Go-ethereum implementation](https://github.com/vulcanize/go-ethereum-EIP3416)
+* [Implementation-specific Magicians thread](https://ethereum-magicians.org/t/eip-1559-go-etheruem-implementation/3918)
+* [First and second-price auctions and improved transaction-fee markets](https://ethresear.ch/t/first-and-second-price-auctions-and-improved-transaction-fee-markets/2410)
+* [The Challenges of Bitcoin Transaction Fee Estimation](https://blog.bitgo.com/the-challenges-of-bitcoin-transaction-fee-estimation-e47a64a61c72)
+* [Gas Now](https://www.gasnow.org/)
+* [Five arguments against fee burning in EIP-1559](https://ethereum-magicians.org/t/five-arguments-against-fee-burning-in-eip-1559/5184)
+* [Robust statistics - Median](https://en.wikipedia.org/wiki/Robust_statistics#Examples)
+* [Evidence of Mempool Manipulation on Black Thursday: Hammerbots, Mempool Compression, and Spontaneous Stuck Transactions](https://www.blocknative.com/blog/mempool-forensics)
+* [Ethereum Network Utilization](https://etherscan.io/chart/networkutilization)
+
```

Comment:
```suggestion
```
Resources is not a valid EIP section.  Also, external links should not be used in EIPs.  Remember, EIPs are technical specifications and not the right place to make arguments over a particular EIP, compare it with other EIPs, etc.  The EIP should be incredibly tightly focused on the technical specification.